processing-service-core/api/swagger/swagger.yaml

```diff
@@ -952,7 +954,231 @@ paths:
           description: The specified Workflow could not be found.
           schema:
             $ref: "#/definitions/errorMessage"    
+  ##########################
+  ##  Tenant Key Store
+  ##########################
+  /tenants/{tenantId}/config/{key}:
+    x-swagger-router-controller: tenant
+    parameters:
+      - name: tenantId
+        in: path
+        type: string
+        description: The ID of the tenant.
+        required: true  
+      - name: key
+        in: path
+        type: string
```

Comment:
Not sure, but `maxLength: 255` should be here based on Micheal's specification [here](https://jira.autonomy.com/browse/SCMOD-3478?focusedCommentId=2028806&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2028806)? So you can test response with 400 status code.